data/mods/Aftershock/items/comestibles/comestibles.json

```
@@ -2,7 +2,7 @@
   {
     "id": "afs_sungel",
     "type": "COMESTIBLE",
-    "name": "sungel",
+    "name": { "str": "sungel" },
```

```suggestion
    "name": { "str_sp": "sungel" },
```
For identical singular and plural forms